### DISCUSSION TITLE
[Features] Order Select, Deselect Characters, and freeze command for slot

### DISCUSSION THREAD

**Discussion by Amidweiz** [ikemen-engine/Ikemen-GO#837] [Feature Request] [answered]
- Order Select
A feature from KOF since KOF 95, this pretty much allows a second confirmation of character order in teams mode rather than needing to know the order initially. Mugen had this feature somewhat where you could do it by holding an direction on the vs screen but it isn't well utilized

- Deselect Characters
Very self explanatory, the ability to deselect a character that you've already selected if you don't want that character.
  
- Freeze command for slot
a reiteration from the discord but

> Another suggestion is to have a parameter on the slot so where it freezes the cursor when certain command is held since I'd rather have the alt character selection with holding S plus up or down rather than needing to use the dedicated tag buttons to achieve this effect.

**Comment by K4thos** [comment, marked as answer]
Order Select is ready for 0.98: https://github.com/K4thos/Ikemen_GO/wiki/Miscellaneous-Info#orderselect
(please remove it from ticket title)

**Comment by MUGEN-Slayer** [comment]
For deselection, the Menu button could function as an "undo" button on the select screen. Instead of returning directly to menu, it could undo the selections and when there is not at least one character selected then it would return to the menu screen. The same way the "X" button works in Japanese games on the Playstation.

**Reply by syzku** [reply]
It doesn't work for me, I returned back to main menu once i press menu button when trying to deselect a character

I use ikemen go version 0.99
